src/bin/sol-fbp-runner/main.c

```diff
@@ -180,6 +408,17 @@ startup(void *data)
         }
     }
 
+    if (args.memory_map_file) {
+#ifdef USE_MEMMAP
+        if (!load_memory_map_file(args.memory_map_file)) {
+            fprintf(stderr, "Could not load memory map file\n");
```

Comment:
`%s` + args.memory_map_file... make messages more verbose. also prefix it with "error:" so tools that offer highlight will change the color of lines (ie: emacs).